src/algorithms/tonal/chordsdetectionbeats.h

```diff
@@ -54,6 +55,7 @@ class ChordsDetectionBeats : public Algorithm {
     void declareParameters() {
       declareParameter("sampleRate", "the sampling rate of the audio signal [Hz]", "(0,inf)", 44100.);
       declareParameter("hopSize", "the hop size with which the input PCPs were computed", "(0,inf)", 2048);
+      declareParameter("chromaPick", "method of calculating singleton chroma for interbeat interval", "{first_beat,interbeat_median}", "interbeat_median");
```

Comment:
We'll need a better explanation of what ```first_peak``` and  ```interbeat_median``` do. Expand the DOC string in the cpp file with a more detailed description.

Reply:
Done. Could you please check?